src/components/poweredByButton/PoweredByButton.js

```diff
@@ -0,0 +1,111 @@
+import React, { useRef, forwardRef, useImperativeHandle } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import Box from '../box';
+import { TextSmall } from '../typography';
+import Icon from '../icon/Icon';
+import TeamleaderText from './teamleaderText.svg';
+import TeamleaderLogo from './teamleaderLogo.svg';
+import TeamleaderLogoBig from './teamleaderLogoBig.svg';
+import theme from './theme.css';
+
+const PoweredByButton = forwardRef(
+  ({ children, href, tint, shape, label, onMouseUp, onMouseLeave, className, ...rest }, ref) => {
+    const buttonRef = useRef();
+    useImperativeHandle(ref, () => buttonRef.current);
+
+    const handleMouseUp = (event) => {
+      blur();
+      if (onMouseUp) {
+        onMouseUp(event);
+      }
+    };
+
+    const handleMouseLeave = (event) => {
+      blur();
+      if (onMouseLeave) {
+        onMouseLeave(event);
+      }
+    };
+
+    const blur = () => {
+      const currentButtonRef = buttonRef.current;
+      if (currentButtonRef.blur) {
+        currentButtonRef.blur();
+      }
+    };
+
+    const classNames = cx(
+      theme['reset-box-sizing'],
+      theme['reset-font-smoothing'],
+      theme['button'],
+      theme[`button-${tint}`],
+      theme[`button-${shape}`],
+      className,
+    );
+
+    const iconProps = tint === 'light' ? { color: 'neutral', tint: 'lightest' } : { tint: 'darkest' };
+
+    return (
+      <Box
+        element="a"
+        href={href}
+        target="_blank"
+        rel="noopener noreferrer"
+        className={classNames}
+        onMouseUp={handleMouseUp}
+        onMouseLeave={handleMouseLeave}
+        ref={buttonRef}
+        {...rest}
+      >
+        {shape !== 'box' && !!children && children}
```

Comment:
In which case would you use your own children here instead of the predefined stuff below here?

Reply:
I put it there in case you would want to translate the `powered by` or so